src/firefly/js/util/WebUtil.js

```diff
@@ -6,6 +6,8 @@ import { forEach, fromPairs, get, has, isArray, isBoolean, isEqual, isFunction,
 import slug from 'slug';
 import shallowequal from 'shallowequal';
 import update from 'immutability-helper';
+import {logger} from './Logger.js';
+
```

Comment:
This is just a quick comment on this PR. (I will look at the PR closer on Monday.)

It would be better, because of future workers, to keep the dependencies of `WebUtil.js` as small as possible. Bringing in `Logger.js` then brings in `moment` with makes the worker much bigger. It would be better to move `logError` or deprecate it and make another version in `Logger.js` than to import logger into `WebUtil.js`

Reply:
I agree.  `WebUtil.logError` is not needed since that functionality is already available in Logger.js.  I would remove logError from WebUtil and replace it with Logger where it was used.  It would involved touching a few more files, though.